packages/augur-ui/src/modules/market/components/common/comments/market-comments.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class MarketComments extends Component {
+  static propTypes = {
```

Comment:
we don't use propTypes anymore we use typescript style. There are a lot of examples.

![image](https://user-images.githubusercontent.com/3970376/65059065-f0a22700-d93a-11e9-8ec5-6b1e11ba08a4.png)

![image](https://user-images.githubusercontent.com/3970376/65059093-ff88d980-d93a-11e9-9f93-155c01a69bea.png)